flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/expressions/ScalarFunctionsTest.scala

```diff
@@ -980,6 +980,61 @@ class ScalarFunctionsTest extends ScalarTypesTestBase {
     testSqlApi("regexp('100-200', CAST(null as VARCHAR))", "NULL")
   }
 
+  @Test
+  def testConv(): Unit = {
+    testAllApis(
```

Comment:
Use the new `BuiltInFunctionTestBase`. It is very powerful as it tests input and output types as well and can be executed in parallel.

Reply:
CC @liuyongvs